### PR TITLE
Prevent some anti-pattern with the Singleton

### PR DESCRIPTION
**Pull request by anthony2727**
@danielbachhuber  do some review in favor of [#17](https://github.com/fusioneng/speed-bumps/issues/17) please. 

Thanks. 
c.c @noppanit

### CODE REVIEW

File: inc/class-speed-bumps.php [hunk 1]
```diff
@@ -12,6 +12,21 @@ public static function get_instance() {
 		return self::$instance;
 	}
 
+	/**
+     * Prevent the creation of a new instance of the "SINGLETON" using the operator 'new' from
+     * outside of this class. 
+     */
```
Comment (danielbachhuber) [inline inc/class-speed-bumps.php]: Can you fix indentation here?

Comment (danielbachhuber) [inline inc/class-speed-bumps.php]: And the other methods as well?

**Comment by anthony2727** [comment]
I got some issues with sublime.Let me know what you think @danielbachhuber

**Comment by danielbachhuber** [comment]
> Let me know what you think

Looks good :+1:

**Comment by anthony2727** [comment]
Thanks @danielbachhuber . I had fun with Travis lol. Great tool.

**Comment by danielbachhuber** [comment]
> I had fun with Travis lol. Great tool.

:)

**Pull request merged**